### PR TITLE
Add test mode message to server info

### PR DESCRIPTION
**Pull request by phil-davis**
So that the web app user will be informed when the server is in random
temperature generating mode.
It should be fine to say nothing when in proper live mode - the normal
end-user should assume it is real unless told otherwise.

### CODE REVIEW

**Comment by phil-davis** [comment]
And the Change Server button is not necessarily below (or to the right) - depends on how the user has resized their screen width.

File: webapp/js/main.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ Temperature.prototype.errorFunc = function(res, textstatus, error, title) {
   this.ready = false;
 
   // say so in server info
-  $('.server-info').html('Not connected to any server. Click the button below to configure.');
+  $('.server-info').html('Not connected to any server. Click the Change Server button to configure.');
```
Comment (swalladge) [inline webapp/js/main.js]: maybe wrap `Change Server` in <code> tags?

File: webapp/js/main.js [hunk 2]
```diff
@@ -233,6 +233,10 @@ function getInitialServerSetup() {
       info += ' Location: <i>' + res.data.location + '</i>';
     }
 
+    if (!res.data.live) {
+      info += '<br /><strong>Server is in test mode generating random temperature data.</strong>';
```
Comment (swalladge) [inline webapp/js/main.js]: methinks <br> is bad practice - inline is fine

File: webapp/js/main.js (hunk 1 above)
Comment (phil-davis) [inline webapp/js/main.js]: Yep, done

**Pull request merged**